README.md

```diff
@@ -152,6 +153,8 @@ This metadata will be saved with the individual files and restored by the [Pull
 | `BITRISE_PUBLIC_INSTALL_PAGE_URL` | Public Install Page's URL, if the *Enable public page for the App?* option was *enabled*. |
 | `BITRISE_PUBLIC_INSTALL_PAGE_URL_MAP` | Public Install Page URLs by the artifact's file path. Only set it if the *Enable public page for the App?* option was *enabled*.  The default format is `KEY1=>VALUE\|KEY2=>VALUE` but is controlled by the `public_install_page_url_map_format` input  Examples:  - $BITRISE_DEPLOY_DIR/ios_app.ipa=>https://ios_app/public/install/page - $BITRISE_DEPLOY_DIR/android_app.apk=>https://android_app/public/install/page\|$BITRISE_DEPLOY_DIR/ios_app.ipa=>https://ios_app/public/install/page |
 | `BITRISE_PERMANENT_DOWNLOAD_URL_MAP` | The output contains permanent Download URLs for each artifact. The URLs can be shared in any communication channel and they won't expire. The default format is `KEY1=>VALUE\|KEY2=>VALUE` where key is the filename and the value is the URL. If you change `permanent_download_url_map_format` input then that will modify the format of this Env Var. You can customize the format of the multiple URLs.  Examples:  - $BITRISE_DEPLOY_DIR/ios_app.ipa=>https://app.bitrise.io/artifacts/ipa-slug/download - $BITRISE_DEPLOY_DIR/android_app.apk=>https://app.bitrise.io/artifacts/apk-slug/download\|$BITRISE_DEPLOY_DIR/ios_app.ipa=>https://app.bitrise.io/artifacts/ipa-slug/download |
+| `BITRISE_ARTIFACT_DETAILS_PAGE_URL` | Details Page's URL.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| `BITRISE_ARTIFACT_DETAILS_PAGE_URL_MAP` | Details Page URLs by the artifact's path. The default format is `KEY1=>VALUE\|KEY2=>VALUE` but is controlled by the `details_page_url_map_format` input  Examples:  - $BITRISE_DEPLOY_DIR/ios_app.ipa=>https://app.bitrise.io/apps/ios_app/installable-artifacts/ipa-slug - $BITRISE_DEPLOY_DIR/android_app.apk=>https://app.bitrise.io/apps/android_app/installable-artifacts/apk-slug\|$BITRISE_DEPLOY_DIR/ios_app.ipa=>https://app.bitrise.io/apps/ios_app/installable-artifacts/ipa-slug                                                                                                                                                                                                 |
```

Comment:
@gaborpongracz What do you think about calling the env `BITRISE_INSTALLABLE_ARTIFACT_DETAILS_PAGE_URL_MAP` it's a bit too long, but at least it highlights that it's only for installable artifacts.